sdk/storage/AzureStorageBlob/AzureStorageBlob.xcodeproj/project.pbxproj

```diff
@@ -11,7 +11,6 @@
 		0A686493233E834D004B7E17 /* AzureStorageBlob.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 0A686489233E834D004B7E17 /* AzureStorageBlob.framework */; };
```

Comment:
Are we running swiftformat automatically as part of the build step?  If not, we should. It could easily go into the `swiftlint.sh` script.

Reply:
Not yet, but we definitely can. Right now there are three call sites where swiftformat replaces the parameter name with `_` because it's not used, but that's either because we have commented out the body of the function or (in one case) I think we have a legitimate bug.

Reply:
Well, swiftlint autocorrect does that too and while it's not wrong it can be annoying. Even so, it wouldn't be hard to replace the _ with something when (or if) we uncomment that code.